services/dashboard/views/apps/streamingapp/streamingapp.html

```diff
@@ -8,13 +8,13 @@
       </h4>
       <h5 ng-bind="app.description"></h5>
     </div>
-    <div class="col-md-2 col-sm-3 col-xs-4">
+    <div class="col-md-2 col-sm-3 col-xs-3">
```

Comment:
For "col-xs-4", "col-xs-8", "col-xs-12", this is how it looks for nexus 7.
![nexux_7](https://cloud.githubusercontent.com/assets/2595532/12382295/66905310-bdd3-11e5-8ee3-80244ab770fc.jpg)